doc/rest.md

```diff
@@ -64,6 +64,7 @@ JSON array of objects representing available layers with following structure:
   - **read**: Array of strings. Names of [users](./models.md#user) and [roles](./models.md#role) with [read access](./security.md#Authorization).
   - **write**: Array of strings. Names of [users](./models.md#user) and [roles](./models.md#role) with [write access](./security.md#Authorization).
 - **bounding_box**: List of 4 floats. Bounding box coordinates [minx, miny, maxx, maxy] in EPSG:3857.
+- **native_crs**: Code of native CRS in form 'EPSG:code'.
```

Comment:
'EPSG:code' -> `"EPSG:<code>"`, e.g. `"EPSG:4326"`.

Add also: Native CRS is CRS of the input data file.